src/rpcserver.cpp

```diff
@@ -27,6 +27,8 @@
 #include <boost/signals2/signal.hpp>
 #include <boost/thread.hpp>
 #include <boost/algorithm/string/case_conv.hpp> // for to_upper()
+#include <boost/algorithm/string/classification.hpp>
```

Comment:
Revert these two lines; this change is not needed for this PR and compiles fine without it.